example/libclang-example/generated_bindings.dart

```diff
@@ -6428,17 +6994,849 @@ class LibClang {
     ffi.Pointer<ffi.NativeFunction<CXFieldVisitor>> visitor,
     ffi.Pointer<ffi.Void> client_data,
   ) {
-    return (_clang_Type_visitFields ??=
-        _lookup<ffi.NativeFunction<_c_clang_Type_visitFields>>(
-                'clang_Type_visitFields')
-            .asFunction<_dart_clang_Type_visitFields>())(
+    return _clang_Type_visitFields(
       T,
       visitor,
       client_data,
     );
   }
 
-  _dart_clang_Type_visitFields? _clang_Type_visitFields;
+  late final _clang_Type_visitFields_ptr =
+      _lookup<ffi.NativeFunction<Native_clang_Type_visitFields>>(
+          'clang_Type_visitFields');
+  late final _dart_clang_Type_visitFields _clang_Type_visitFields =
+      _clang_Type_visitFields_ptr.asFunction<_dart_clang_Type_visitFields>();
+
+  late final addresses = _SymbolAddress(this);
+}
+
+class _SymbolAddress {
```

Comment:
Maybe `_SymbolAddresses` because it holds all of them?

Reply:
Done.